src/component/Proposals/LiveChat/LiveChat.js

```diff
@@ -19,6 +20,7 @@ const match = {
 export default function LiveChat({ ual, resolution, encryptionKey }) {
   let [newMessage, setNewMessage] = useState("");
   let [messages, setMessages] = useState([]);
+  const theme = useTheme();
 
   async function sendMessage() {
```

Comment:
Me who haven't seen in but it might be better to make it an arrowless function. Also performance wise, we should put the function on top because the function is defined everytime the component render.

Reply:
const  sendMessage =  async () => {}